src/rdkafka_aux.c

```diff
@@ -97,3 +97,76 @@ void rd_kafka_topic_result_destroy (rd_kafka_topic_result_t *terr) {
 void rd_kafka_topic_result_free (void *ptr) {
         rd_kafka_topic_result_destroy((rd_kafka_topic_result_t *)ptr);
 }
+
+rd_kafka_resp_err_t
+rd_kafka_group_result_error (const rd_kafka_group_result_t *groupres) {
+        return groupres->err;
```

Comment:
Have this return an `rd_kafka_error_t *` object (or NULL) and then get rid of error_string below.